### PR TITLE
Medical - Allow unconscious unit in Taru pods

### LINKED ISSUE #8167: Unconscious incompatible with Taru Medical/Transport Pod
**Mods (complete and add to the following information):**
- **Arma 3:** `2.02 stable`
- **CBA:** `3.15.3 workshop`
- **ACE3:** `3.13.6 workshop`

**Description:**

**Steps to reproduce:**

1. Setup: create mission with player, patient unit, Taru Medical Pod, Taru Transport Pod, play
2. Knock out patient: Create Zeus, ACE Medical > Toggle Unconscious
3. Try to load:
    3a. Drag patient into pod: patient on ground
    3b. Exit Zeus, interact with patient > Load Patient > Pod: patient on ground
4. Bonus: 
    4a. Place conscious unit in pod, works.
    4b. Drop Toggle Unconscious module on pod: patient unconscious and kicked out of pod

**Expected behavior:**
The unconscious unit should end up in one of the seats in the pod.

**Where did the issue occur?**
- Editor (Singleplayer)

**Log Files:**
- https://pastebin.com/8Bs4dX9A

**Additional context:**
`_patient moveInCargo _pod` fails

**Screenshots:**
https://youtu.be/4B2mHngw70Y

### PR DESCRIPTION
**Pull request by ampersand38**
**When merged this pull request will:**
- Check vehicle type for Taru pods and don't eject unit from them
- Close #8167

### CODE REVIEW

File: addons/medical_engine/functions/fnc_setUnconsciousAnim.sqf [hunk 1]
```diff
@@ -27,7 +27,7 @@ _unit setUnconscious _isUnconscious;
 
 if (_isUnconscious) then {
     // eject from static weapon
-    if (vehicle _unit isKindOf "StaticWeapon") then {
+    if (vehicle _unit isKindOf "StaticWeapon" && {!(vehicle _unit  isKindOf "Pod_Heli_Transport_04_crewed_base_F")}) then {
```
Comment (veteran29) [inline addons/medical_engine/functions/fnc_setUnconsciousAnim.sqf:30]:
```suggestion
    if (vehicle _unit isKindOf "StaticWeapon" && {!(vehicle _unit isKindOf "Pod_Heli_Transport_04_crewed_base_F")}) then {
```

**Review by veteran29 — COMMENTED** [review]
Double space.

**Review by veteran29 — APPROVED** [review]

**Review by kymckay — APPROVED** [review]
Kind of hate how hardcoded it feels, but probably best we can get for this

**Comment by ampersand38** [comment]
Yeah, it's also already here as well https://github.com/acemod/ACE3/blob/f85632cbbd7ae69eabbd08e6b62c28bb1ae318f9/addons/common/functions/fnc_nearestVehiclesFreeSeat.sqf#L22

**Pull request merged**